### ISSUE TITLE
Model acc loss after BN fused

### ISSUE THREAD

**Issue by Jzz24** [jakc4103/DFQ#20]
Thx for your great job ! but I have some questions about your code:
1. the fused model(mobilenetv2) loss 1% acc compare to the fp model . In the main_cls.py , you do switch_layers() before merge_batchnorm(), after the switch_layer() the QuantNconv2d() quantize the input to 8bits by default ? I'm confused about it 
2. in the merge_batchnorm(), you save the torch.abs( bn gamma) in the 'fake weight' buffer, why not bn gamma ?

**Comment by jakc4103** [comment]
If you want to evaluate in FP32, use torch.nn.Conv2d. QuantNConv2d, QauntConv2d, and QConv2d is meant to be used to evaluate quantized model.

In pytorch, BN weight refer to the scaling parameter gamma, they are basically the same.

**Comment by Jzz24** [comment]
BN weight refer to the scaling parameter gamma, but the gamma may be negative, I want to know , why you save 'fake weights' as abs(BN weights)?

**Comment by jakc4103** [comment]
They are used to compute bias correction as original paper stated: the bn weight and bias can be seen as the mean and std of feature distribution.
